Samples/ResourceManagement/cpp-console-unpackaged/console_unpackaged_app.cpp

```diff
@@ -58,11 +59,11 @@ int wmain(int argc, wchar_t* argv[])
         return 1;
     }
 
-    // Initialize dynamic dependencies so we can consume the Project Reunion APIs in the Project Reunion framework package from this unpackaged app. 
-    HRESULT loadProjectReunionHr = LoadProjectReunion();
-    if (FAILED(loadProjectReunionHr))
+    // Initialize dynamic dependencies so we can consume the WindowsAppSDK APIs in the WindowsAppSDK framework package from this unpackaged app. 
+    HRESULT loadWindowsAppSDKHr = LoadWindowsAppSDK();
+    if (FAILED(loadWindowsAppSDKHr))
     {
-        std::wcout << "Could not load Project Reunion!" << std::endl;
+        std::wcout << "Could not load WindowsAppSDK!" << std::endl;
```

Comment:
>WindowsAppSDK [](http://example.com/codeflow?start=38&length=13)

A comment for the change overall: Windows App SDK in text (whether in error messages or comments or whatever else) shouldn't be in CamelCase. The name of the product is "Windows App SDK". The CamelCase version can be for code of course.